finetuning/generalists/training/electron_microscopy/mito_nuc/train_mito_nuc_em_generalist.py

```diff
@@ -0,0 +1,137 @@
+import os
+import argparse
+
+import torch
+
+from torch_em.model import UNETR
+from torch_em.loss import DiceLoss, DiceBasedDistanceLoss
+
+import micro_sam.training as sam_training
+from micro_sam.util import export_custom_sam_model
+
+from obtain_mito_nuc_em_datasets import get_generalist_mito_nuc_loaders
+
+
+def finetune_mito_nuc_em_generalist(args):
+    """Code for finetuning SAM on mitochondria and nuclei electron microscopy datasets"""
+    # override this (below) if you have some more complex set-up and need to specify the exact gpu
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+
+    # training settings:
+    model_type = args.model_type
+    checkpoint_path = None  # override this to start training from a custom checkpoint
+    patch_shape = (1, 512, 512)  # the patch shape for training
+    n_objects_per_batch = 25  # this is the number of objects per batch that will be sampled
+    freeze_parts = None  # override this to freeze one or more of these backbones
+
+    # get the trainable segment anything model
+    model = sam_training.get_trainable_sam_model(
+        model_type=model_type,
+        device=device,
+        checkpoint_path=checkpoint_path,
+        freeze=freeze_parts
+    )
+    model.to(device)
+
+    # let's get the UNETR model for automatic instance segmentation pipeline
+    unetr = UNETR(
+        backbone="sam",
+        encoder=model.sam.image_encoder,
+        out_channels=3,
+        use_sam_stats=True,
+        final_activation="Sigmoid",
+        use_skip_connection=False
```

Comment:
It's probably best to set `resize_input=True` here. (This is True by default now, but it's good to be explicit here.)

Reply:
Makes sense. Updated this as wlel. Thanks!